docs/en/sql-reference/functions/tuple-map-functions.md

```diff
@@ -66,6 +66,40 @@ Result:
 
 -   [Map(key, value)](../../sql-reference/data-types/map.md) data type
 
+## mapFromArrays
+
+mapFromArrays merges an array of keys and an array of values into a Map.
```

Comment:
`array of keys` --> `[Array](../sql-reference/data-types/array.md) of keys`

`array of values` --> `[Array](../sql-reference/data-types/array.md) of values`

`Map` --> `[Map(key, value)](../../sql-reference/data-types/map.md)`